cmd/manager/scap_test.go

```diff
@@ -316,6 +316,20 @@ var _ = Describe("Testing filtering", func() {
 				Expect(filterErr).Should(MatchError(MoreThanOneObjErr))
 			})
 		})
+		Context("Piped Filtering", func() {
+			var rawmc []byte
+			BeforeEach(func() {
+				nsFile, err := os.Open("../../tests/data/empty_machineconfig.json")
+				Expect(err).To(BeNil())
+				var readErr error
+				rawmc, readErr = io.ReadAll(nsFile)
+				Expect(readErr).To(BeNil())
+			})
+			It("skips filter piping errors", func() {
```

Comment:
nit: we could include the nil aspect into the test name here, so it's more specific about what case we're testing for.

```go
It("gracefully handles nil item lists", func() {
```